xsd/netex-nl-basic.xsd

```diff
@@ -143,7 +140,7 @@ De waarde 'any' van attribuut 'version' wordt geinterpreteerd als 'de van het bo
 		</xsd:annotation>
 		<xsd:attribute name="id" type="ObjectIdType" use="required" form="unqualified"/>
 		<xsd:attribute name="modification" type="ModificationEnumeration" use="optional" default="new" form="unqualified"/>
-		<xsd:attribute name="version" type="VersionIdType" use="optional" default="any" form="unqualified"/>
+		<xsd:attribute name="version" type="VersionIdType" use="required" form="unqualified"/>
```

Comment:
Bewuste keuze default hier weg te halen?

Reply:
ja: als het attribuut required is, overschrijf je altijd de default waarde, dus is die niet nodig